server/src/routes/api/v1/categoriesRouter.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import {Category} from '../../../models/index.js'
+
+const categoriesRouter = new express.Router()
+
+categoriesRouter.get('/', async (req, res) => {
+  try {
+    const categories = await Category.query()
```

Comment:
Even though a category just has a name, I would encourage you to create a serializer to remove things like timestamps!